modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -60,7 +60,7 @@ class ColumnStoreDataset:
     """
 
     def __init__(self, path, storage_options):  # noqa : PR01
-        self.path = path
+        self.path = path.__fspath__() if isinstance(path, os.PathLike) else path
```

Comment:
Why use `__fspath__`? It looks protected.

Reply:
That's the only way to get the path. `pandas` does the same thing.